api/v1/keycloakcomponent_types.go

```diff
@@ -33,6 +33,9 @@ type KeycloakComponentSpec struct {
 	ParentRef *ParentComponent `json:"parentRef,omitempty"`
 
 	// Config is a map of component configuration.
+	// Map key is a name of configuration property, map value is an array value of configuration properties.
```

Comment:
Change the commit message to our approach:
`feat: Allow secret references in KeycloakRealmComponent (#issue_number)`
Example: feat: Allow secret references in KeycloakRealmComponent (#123)

Reply:
I created issue #30 to have something to refer to. Now reflected in commit message